HermaFx.DataAnnotations/DistinctElementsAttribute.cs

```diff
@@ -66,7 +66,7 @@ protected override ValidationResult IsValid(object value, ValidationContext cont
 				return ValidationResult.Success;
 			}
 
-			return new ValidationResult(FormatErrorMessage(UniqueProperty));
+			return new ValidationResult(string.Format(ErrorMessage, UniqueProperty, context.MemberName));
```

Comment:
I think you need to actually use FormatErrorMessage, as it takes care of using the fixed message passed as ErrorMessage, but also supports ErrorMessageResourceName (or whtaever it is named), etc.

Regarding passing additional format parameters to format error message.. I dont know / remember now the appropiate way, but we may want to take a look a source code or examples of other attribures with similiar issues.

Reply:
could it be by overriding the FormatErrorMessage method?
```csharp
public override string FormatErrorMessage(string name)
{
    return string.Format(ErrorMessageString, UniqueProperty, name);
}
```

Reply:
I guess so..

Reply:
its already changed